contracts/implementation/PoolFactory.sol

```diff
@@ -212,6 +224,17 @@ contract PoolFactory is IPoolFactory, Ownable {
         poolKeeper = IPoolKeeper(_poolKeeper);
     }
 
+    /**
+     * @notice Sets the address of the associated `AutoClaim` contract
+     * @param _autoClaim Address of the `AutoClaim`
+     * @dev Throws if provided address is null
+     * @dev Only callable by the owner
+     */
+    function setAutoClaim(address _autoClaim) external override onlyOwner {
```

Comment:
i don't see this function called in the constructor? Does this mean deployPool is unusable unless setAutoClaim is called? If thats the case, we should probably create the autoclaim contract and set it in constructor.

Reply:
This is following the same pattern as what is used with `poolKeeper`. I agree that your suggestion makes sense, so I've added a ticket to do it for both poolKeeper and autoClaim: https://github.com/tracer-protocol/perpetual-pools-contracts/issues/258